boards/thingpulse_epulse_feather_c6.json

```diff
@@ -0,0 +1,30 @@
+{
+  "build": {
+    "core": "esp32",
+    "f_cpu": "160000000L",
+    "f_flash": "80000000L",
+    "flash_mode": "qio",
+    "mcu": "esp32c6",
```

Comment:
Where are the `extra_flags` and `ldscript` entries for Arduino framework?

Reply:
I added `extra_flags`, but the only example I have for esp32-c6 is the espressif modules and they don't have the `ldscript` value set